README.md

```diff
@@ -1 +1,73 @@
 # ArDrive Price Calculator
+
+## Developer Setup
+
+Follow these steps to get the developer environment up and running:
+
+### Install Yarn 2
+
+The ArDrive Price Calculator uses Yarn 2, so install the latest version with the [yarn installation instructions][yarn-install]. In most cases:
+
+```shell
+# Brew:
+brew install yarn
+
+# Or with NPM:
+npm install -g yarn
+```
+
+We also use husky. To enable hooks locally, you will need to run:
+
+```shell
+yarn husky install
+```
+
+### NVM
+
+This repository uses NVM and an `.nvmrc` file to lock the Node version to the current version used by `ardrive-core-js`.
+
+Note for Windows: **We recommend using WSL** for setting up NVM on Windows using the [instructions described here][wsl-install]
+
+Follow these steps to get NVM up and running on your system:
```

Comment:
It might be helpful to explain which steps are to be performed once and which are to be used every time you start a new terminal session.